yaml/boundaries.yaml

```diff
@@ -189,6 +213,22 @@ filters:
       <<: *output_properties
       kind: disputed_reference_line
       disputed: True
+      recognized_by: { col: recognized_by }
+      tz_admin_level: {col: admin_level }
+    table: osm
+  # These are kind=disputed_reference_lines that, for viewpoint XX in disputed_by,
+  # becomes kind:xx=unrecognized_disputed_reference_line
+  - filter: {admin_level: ['2','4','6'], boundary: disputed, geom_type: line}
+    min_zoom: 8
+    output:
+      <<: *output_properties
+      kind: disputed_reference_line
```

Comment:
NE at the low zooms just calls this `kind: disputed`, let's discuss IRL now we have a build together enough where it's easier to reason about this.

Reply:
admin_level: 2 > kind: disputed
admin_level: 4 > kind: disputed_region
admin_level: 6 > kind: disputed_county